### PR TITLE
Update standard deduction page

### PR DESCRIPTION
**Pull request by kategreenUSDS**
## What's in here
* Remove bypass expenses code. These don't appear in the designs anymore 
* Update page contents to reflect design updates

## Screens
Screenshot 2024-08-28 at 1 25 45 PM
Screenshot 2024-08-28 at 1 26 04 PM

## What's missing
* The accordion is stub; we still need content for this
* Edit functionality, ability to review choice of std deduction or not
* We aren't yet tracking a user's state (work tbd), so i left a placeholder `[your state's]` in the title. It should be Very Obvious to fix this later on

## Review requests
Nothing specific this time. I changed some routing and removed a page and its test

### CODE REVIEW

File: app/[locale]/ledger/expense/snap/page.tsx [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
+import { selectRecommendStandardDeduction } from "@/lib/store"
 import VerifyNav from "@/app/components/VerifyNav"
 
 export default function Page() {
     const { t } = useTranslation()
     const router = useRouter()
+    const dispatch = useAppDispatch()
+    const benefits = useAppSelector(state => selectBenefits(state))
+    const expenseTotal = useAppSelector(state => selectExpenseTotal(state))
+    const reccommendStandardDeduction = useAppSelector(state => selectRecommendStandardDeduction(state))
```
Comment (joeyg) [inline app/[locale]/ledger/expense/snap/page.tsx:20]: nit: typo with recommend

File: app/[locale]/ledger/expense/snap/page.test.tsx [hunk 2]
```diff
@@ -1,48 +1,61 @@
...
     it('shows header', () => {
-        expect(screen.getByTestId('expense-snap-header')).toBeDefined()
+        expect(screen.getByTestId('snap_deduction_header')).toBeDefined()
     })
 
-    it('navigates to review screen if no is selected', async () => {
-        fireEvent.click(screen.getByTestId("no_radio"))
+    it('navigates to review screen if take deduction selected', async () => {
+        const radio: HTMLInputElement = screen.getByTestId("take_deduction_radio")
+        fireEvent.click(screen.getByText(/Take the standard deduction/i))
```
Comment (joeyg) [inline app/[locale]/ledger/expense/snap/page.test.tsx:45]: Are we able to do this without searching for the words so the tests are not relying on the copy?

File: app/[locale]/ledger/expense/snap/page.tsx [hunk 3]
```diff
@@ -22,42 +30,85 @@ export default function Page() {
...
-                                <span className="usa-hint">{t('expenses_snap_standard_subheader')}</span>
+                                <h3 className="margin-bottom-2" data-testid="snap_deduction_header">{t('snap_deduction_header', {amount: expenseTotal})}</h3>
+
+                                <span className="usa-hint">{t('snap_deduction_subheader')}</span>
+
+                                <Accordion multiselectable={true} className="margin-top-2" items={items} />
+
                                 <Controller
-                                    name="snapRadio"
+                                    name="take_deduction"
```
Comment (joeyg) [inline app/[locale]/ledger/expense/snap/page.tsx:82]: It looks like both radio buttons can be selected (screenshot)

Screenshot 2024-09-05 at 9 19 26 AM

File: app/[locale]/ledger/expense/snap/page.test.tsx [hunk 4]
```diff
@@ -1,48 +1,61 @@
...
     afterEach(cleanup)
 
     it('shows header', () => {
-        expect(screen.getByTestId('expense-snap-header')).toBeDefined()
+        expect(screen.getByTestId('snap_deduction_header')).toBeDefined()
     })
 
-    it('navigates to review screen if no is selected', async () => {
-        fireEvent.click(screen.getByTestId("no_radio"))
+    it('navigates to review screen if take deduction selected', async () => {
```
Comment (joeyg) [inline app/[locale]/ledger/expense/snap/page.test.tsx:43]: Can we add a test for if the other option is selected

**Review by joeyg — COMMENTED** [review]
Great work! I left a few comments about the tests and I think the radio buttons need to have the same name attribute.

File: app/[locale]/ledger/expense/snap/page.test.tsx (hunk 2 above)
Comment (kategreenUSDS) [reply app/[locale]/ledger/expense/snap/page.test.tsx:45]: i think it's a good idea to eliminate the words in all the tests. i haven't gotten to it yet though

File: app/[locale]/ledger/expense/snap/page.tsx (hunk 3 above)
Comment (kategreenUSDS) [reply app/[locale]/ledger/expense/snap/page.tsx:82]: ooh thank you, let me cut a bug. i did a bunch of radio button work in my current ticket so i have radio buttons down haha

**Review by joeyg — APPROVED** [review]

File: app/[locale]/ledger/expense/snap/page.tsx (hunk 3 above)
Comment (kategreenUSDS) [reply app/[locale]/ledger/expense/snap/page.tsx:82]: i'm gonna catch this (one line) fix in my next PR. thanks for finding this

**Pull request merged**